packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -668,6 +683,59 @@ class Auth {
         return await this.loginGuestUser()
     }
 
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: authorizeIDP.
+     *
+     */
+    async authorizeIDP(parameters: AuthorizeIDPParams) {
+        const privateClient = !!this.clientSecret
+        const redirectURI = this.redirectURI
+        const usid = this.get('usid')
+        const {url, codeVerifier} = await helpers.authorizeIDP(
+            this.client,
+            {
+                redirectURI,
+                hint: parameters.hint,
+                ...(usid && {usid})
+            },
+            privateClient
+        )
+        window.location.assign(url)
```

Comment:
This `Auth` class is designed to work isomorphically both on node and browser. It would be nice if we can wrap it in case this is accidentally ran on the server side.
```suggestion
    if (window) {
        window.location.assign(url)
    } else {
        console.warn('Warning: something might be wrong, this client side method is invoked on the server.')
    }
```